src/transformers/models/whisper/convert_openai_to_hf.py

```diff
@@ -151,7 +155,7 @@ def convert_openai_whisper_to_tfms(checkpoint_path, pytorch_dump_folder_path):
         encoder_layers=dimensions["n_audio_layer"],
         encoder_attention_heads=dimensions["n_audio_head"],
         decoder_layers=dimensions["n_text_layer"],
-        decoder_attention_heads=dimensions["n_text_state"],
```

Comment:
Surprised we managed to convert the original checkpoints with this bug @ArthurZucker 🤔 The state dicts surely won't have matched? Maybe we hardcoded this before?

Reply:
Yeah, but I think I hardcoded the values when converting and then later on made it automatic. I checked by actually re-running the script and seeing that this was a nice type 🤣 but good sign that no one else tried to convert the checkpoints !

Reply:
Not sure about the history behind this. But looking at the `blame`, the script was correct once: https://github.com/huggingface/transformers/blob/a6361d447f0d66aa8a1ec81e8721b7376b776846/src/transformers/models/whisper/convert_openai_whisper_to_tfms.py#L99

Then it was deleted and recovered in: https://github.com/huggingface/transformers/pull/20600

That's where the problem seems to come from. So I would say the original script you used worked.

Reply:
Nice digging! Yep I think I uploaded an old version late by a few commits